Makefile

```diff
@@ -108,6 +109,11 @@ test: check_conda
 	@echo "Running tests (skip slow and online tests) ..."
 	@-bash -c "source $(ANACONDA_HOME)/bin/activate $(CONDA_ENV);pytest -v -m 'not slow and not online'"
 
+.PHONY: test_nb
+test_nb:
+	@echo "Running notebook-based tests"
+	@bash -c "source $(ANACONDA_HOME)/bin/activate $(CONDA_ENV);env FINCH_WPS_URL=$(RAVEN_WPS_URL) pytest --nbval $(CURDIR)/docs/source/notebooks/ --sanitize-with $(CURDIR)/docs/source/output_sanitize.cfg --ignore $(CURDIR)/docs/source/notebooks/.ipynb_checkpoints"
```

Comment:
`FINCH_WPS_URL=$(RAVEN_WPS_URL)` -> `FINCH_WPS_URL=$(FINCH_WPS_URL)`

Reply:
Sorry, this PR is a mess. I thought it was done, but it's a WIP. Will ping you when it's working.